config/manager/kustomization.yaml

```diff
@@ -18,5 +18,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/kubevirt/ssp-operator
-  newTag: latest
+  newName: registry:5000/kubevirt/ssp-operator
```

Comment:
I would prefer to not commit these changes

Reply:
this got changed with `make vendor generate manifests`

Reply:
This is my fault, the `kubevirt` and `kubevirtci` makefile targets leave artefacts like this around in `config`. I'll raise this as an issue tomororw and will fix this up but for now please drop them from this PR.

Reply:
done